pkg/subnet/public.go

```diff
@@ -105,6 +108,135 @@ func (d *PublicDeployer) AddValidator(
 	return false, tx, nil
 }
 
+func (d *PublicDeployer) CreateAssetTx(
+	subnetID ids.ID,
+	tokenName string,
+	tokenSymbol string,
+	denomination byte,
+	initialState map[uint32][]verify.State,
+) (ids.ID, error) {
+	wallet, err := d.loadWallet(subnetID)
+	if err != nil {
+		return ids.Empty, err
+	}
+
+	if d.usingLedger {
+		ux.Logger.PrintToUser("*** Please sign remove validator hash on the ledger device *** ")
+	}
+
+	id, err := wallet.X().IssueCreateAssetTx(tokenName, tokenSymbol, denomination, initialState)
+	if err != nil {
+		return ids.Empty, err
+	}
+	ux.Logger.PrintToUser("Create Asset Transaction successful, transaction ID: %s", id)
+	ux.Logger.PrintToUser("Now exporting asset to P-Chain ...")
+	return id, err
+}
+
+func (d *PublicDeployer) ExportToPChainTx(
+	subnetID ids.ID,
+	subnetAssetID ids.ID,
+	owner *secp256k1fx.OutputOwners,
+	maxSupply uint64,
+) (ids.ID, error) {
+	wallet, err := d.loadWallet(subnetID)
+	if err != nil {
+		return ids.Empty, err
+	}
+
+	if d.usingLedger {
+		ux.Logger.PrintToUser("*** Please sign remove validator hash on the ledger device *** ")
+	}
+
+	id, err := wallet.X().IssueExportTx(ids.Empty,
+		[]*avax.TransferableOutput{
+			{
+				Asset: avax.Asset{
+					ID: subnetAssetID,
+				},
+				Out: &secp256k1fx.TransferOutput{
+					Amt:          maxSupply,
+					OutputOwners: *owner,
+				},
+			},
+		})
+	if err != nil {
+		return ids.Empty, err
+	}
+	ux.Logger.PrintToUser("Export to P-Chain Transaction successful, transaction ID: %s", id)
```

Comment:
other option is to check for err nil and then print this message, and just return id, err afterwards